src/pages/en/guides/integrations-guide.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Using Integrations
+---
+
+**Astro Integrations** can extend Astro with new functionality and behaviors. Astro's Integration API is inspired by Rollup and builds on top of Vite's already impressive plugin API. Taken together, this system gives you access to configure just about everything in Astro, from high-level Astro dev/build features to low-level file loading & build logic. 
+
```

Comment:
Can we have some examples of integrations here?  (I might not even know exactly what sorts of things we're talking about at this point. React? Tailwind? any old npm package? Snipcart? a CMS?)

Reply:
Done!